DEVELOP.rst

```diff
@@ -10,21 +10,24 @@ You will need Python 3.5 installed.
 Setup
 =====
 
-Bootstrap the project::
+Installing node locally::
 
-    $ python bootstrap.py
-    $ bin/buildout -N
+    $ /usr/bin/python3 -m venv env
+    $ source env/bin/activate
```

Comment:
```suggestion
```

there's no need to activate. you can just call the binaries directly from the env